spec/unit/indirector/catalog/compiler_spec.rb

```diff
@@ -150,15 +150,9 @@ def a_request_that_contains(facts)
       @compiler.extract_facts_from_request(request).should be_nil
     end
 
-    it "deserializes the facts and timestamps them" do
-      @facts.timestamp = Time.parse('2010-11-01')
+    it "should deserialize the facts" do
       request = a_request_that_contains(@facts)
-      now = Time.parse('2010-11-02')
-      Time.stubs(:now).returns(now)
-
       facts = @compiler.extract_facts_from_request(request)
-
-      facts.timestamp.should == now
```

Comment:
@joshcooper ah yeah, I was going off of the original test, which essentially just did this (I had the same thought :)) I also toyed with the timestamp idea. I can add that in.